public/css/jumbotron-narrow.css

```diff
@@ -74,6 +73,29 @@ body {
   margin-top: -75px;
 }
 
+.campaign-form {
+  font-size: 16px;
+  background-color: white;
+  color: #35A5B3;
+  margin-left: 50px;
+  margin-right: 70px;
```

Comment:
Something is indeed screwy. Maybe there is a div near the top of the dom tree that has a right padding or margin. Would you look into that. Its weird because the header doesn't reach the right edge of the view window:
![image](https://cloud.githubusercontent.com/assets/970117/22413035/229f7cde-e669-11e6-8557-683cb9e491bc.png)

Also note that the script textbox is wider than the others. This is iPhone 6 view on desktop chrome.